inspirehep/modules/workflows/models.py

```diff
@@ -50,3 +58,16 @@ def save(self):
         """Save object to persistent storage."""
         with db.session.begin_nested():
             db.session.add(self)
+
+
+class WorkflowsPendingRecord(db.Model):
+
+    __tablename__ = "workflows_pending_record"
+
+    workflow_id = db.Column(db.Integer, primary_key=True, nullable=False)
+    record_id = db.Column(db.Integer, nullable=False)
+
+    def save(self):
```

Comment:
I see it is also on the model above, but I am not sure why this is needed. On models in other modules we don't have a dedicated `save` function - https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/models.py